app/javascript/src/index.js

```diff
@@ -89,8 +98,8 @@ switch(controllerAndAction()) {
     break;
 
   default:
-       console.log(controllerAndAction());
-       Controller = DefaultController;
+    //console.log(controllerAndAction());
```

Comment:
cheeky comment

Reply:
yeah, I lef tthis deliberately as its helpful to just uncomment when you need debugging